collect_app/src/androidTest/java/org/odk/collect/android/utilities/BitmapScaledToDisplayTest.java

```diff
@@ -0,0 +1,133 @@
+package org.odk.collect.android.utilities;
+
+import android.graphics.Bitmap;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.odk.collect.android.application.Collect;
+
+import java.io.File;
+
+import static org.junit.Assert.assertEquals;
+
+
+@RunWith(AndroidJUnit4.class)
+public class BitmapScaledToDisplayTest {
+
+    @Test
+    public void scaleDownBitmapWhenPossible() {
```

Comment:
Here’s one way to reduce code duplication:

![screen shot 2018-03-18 at 7 30 04 pm](https://user-images.githubusercontent.com/55099/37575190-0cd8eaf6-2ae3-11e8-9706-bb8ecf68cdf2.png)

https://github.com/dcbriccetti/collect/commit/fecde6c3c3f9eef6637401b52abf728969260630

Reply:
Done